src/server/bqjob/job.go

```diff
@@ -140,6 +140,32 @@ func (job *Job) getResultTable() (*bigquery.Table, error) {
 	return table, nil
 }
 
+func (job *Job) GetResultTableForScript() (*bigquery.Table, error){
+
+	client, err := bigquery.NewClient(job.GetCtx(), os.Getenv("DEKART_BIGQUERY_PROJECT_ID"))
+
+
+	jobFromJobId, err := client.JobFromID(job.GetCtx(), job.bigqueryJob.ID())
+	if err != nil{
+		return nil, err
+	}
+
+	cfg, err := jobFromJobId.Config()
+	queryConfig := cfg.(*bigquery.QueryConfig)
```

Comment:
It is possible that type assertion was not successful, and we will have segfault on the next steps. Please test if type assertion was successful and handle unsuccessful assertion.

```golang
queryConfig, ok := cfg.(*bigquery.QueryConfig)
if !ok {
  // handle error
}
```

Reply:
Added in proper error in case of an unexpected configuration.  
https://github.com/dekart-xyz/dekart/pull/71/commits/12b8ffc823aa25e44a7d3bc0c0ae4fd13b128daf